src/univers/version_range.py

```diff
@@ -898,6 +898,16 @@ def from_native(cls, string):
             constraints.append(constraint)
         return cls(constraints=constraints)
 
+    @classmethod
+    def from_sequence(cls, sequence):
```

Comment:
Could you add a docstring?
what is this doing? what is the input and what is this returning?
May be `Return a version range from a list of version strings.`

Also is `from_sequence` the best name? what is the use case here? I would say this is to create a range from a list of versions? what about `from_versions`